<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferences</class>
 <widget class="QDialog" name="preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>404</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>646</width>
    <height>404</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>646</width>
    <height>404</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/Pictures/wallpaper.png</normaloff>:/icons/Pictures/wallpaper.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>431</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="toolTip">
       <string>Reset preferences to the default onces</string>
      </property>
      <property name="text">
       <string>Reset Preferences</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>&amp;Close</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="toolTip">
       <string>Save and close the dialog</string>
      </property>
      <property name="text">
       <string>&amp;Save</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>491</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>431</width>
       <height>108</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Change Desktop Wallpaper ( Once )</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Start Changing Desktop Wallpapers</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Auto-enable Live Earth Wallpaper</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>Auto-enable Picture Of The Day</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>67</y>
       <width>511</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Enable startup option</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Startup Settings</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>511</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QComboBox" name="combostyle_2">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>250</y>
       <width>451</width>
       <height>27</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Arabic (su)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bosnian (bs)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Croatian (hr)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Danish (da)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">English (en)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>French (fr)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>German (de)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Greek (el)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hebrew (he)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Italian (it)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Netherlands (nl)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Norwegian (nb)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Polish (pl)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Portuguese (pt)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Russian (ru)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Slovak (sk)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spanish (es)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Swedish (sv)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Turkish (tr)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>461</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Language: ( You will need to restart Wallch )</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combostyle_3">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>300</y>
       <width>451</width>
       <height>27</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>Do not keep history</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Simple (month and day of month)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Advanced (month, day,hour,minute)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Very analytical (month, day,hour,minute,second)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>461</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>History:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>461</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Default Style of wallpaper images:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combostyle">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>200</y>
       <width>451</width>
       <height>27</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>Tile</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Zoom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Center</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Scale</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Stretch</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Span</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>General Settings</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>511</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="rotate_checkBox">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>137</y>
       <width>461</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotate images on add, based on Exif data (will alter image)</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>60</y>
       <width>461</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Minimize to tray ( Not recommened for Unity )</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hideapp_checkbox">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Hide app when 'Start' is pressed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="unity_prog_checkbox">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Unity ProgressBar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QWidget" name="layoutWidget4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>461</width>
       <height>149</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Notification on Wallpaper change</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="playsoundcheckBox">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Play sound everytime a wallpaper changes</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="customsoundcheckBox">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Use custom sound</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browsesoundButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Click to select your own sound</string>
        </property>
        <property name="text">
         <string>(None)</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/Pictures/audio-x-generic.png</normaloff>:/icons/Pictures/audio-x-generic.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Stop all possible sounds</string>
        </property>
        <property name="text">
         <string>Stop All Sounds</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>511</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Notification Settings</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>14</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Startup</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>107</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>General</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Notifications</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
